tests/components/Card/card.test.js

```diff
@@ -98,24 +103,6 @@ describe('Card component', () => {
     expect(eventWatch).toHaveBeenCalled();
   });
 
-  it('adds fhirServiceUrl, patientId, and smart_messaging_origin params to the smart link if launched openly', () => {
```

Comment:
This practice of passing `patientId` and `fhirServiceUrl` can be convenient for debugging but causes confusion about how a SMART app launch is expected to work -- better to focus on the standards-compliant SMART launch flow.

Reply:
The SMART JS client has support for these parameters, but other SMART client libraries don't (e.g. our Python library). Also the docs note: "These should ONLY be used in development.", but the CDS Hooks sandbox silently enters this dev mode and makes it look (to a sandbox user) like a standard SMART launch.

Let's think about how to add this kind of functionality back more carefully, without labeling it as a "SMART App Launch". To be clear, this only impacts developers testing the SMART app launch -- and it's important that those developers understand what the SMAT app launch is (and isn't). For other purposes they can just use a non-SMART link card.